lib/index.d.ts

```diff
@@ -0,0 +1,279 @@
+/**
+ * Debounce fn, calling it only once if
+ * the given time elapsed between calls.
+ *
+ * @param  {Function} fn
+ * @param  {Number} timeout
+ *
+ * @return {Function} debounced function
+ */
+export function debounce(fn: Function, timeout: number): (...args: any[]) => void;
+
+/**
+ * Bind function against target <this>.
+ *
+ * @param  {Function} fn
+ * @param  {Object}   target
+ *
+ * @return {Function} bound function
+ */
+export function bind(fn: Function, target: object): Function;
+
+/**
+ * Copy the values of all of the enumerable own properties from one or more source objects to a
+ * target object. Returns the target object.
+ * @param target The target object to copy to.
+ * @param source The source object from which to copy properties.
+ */
+export function assign<T, U>(target: T, source: U): T & U;
+
+/**
+ * Copy the values of all of the enumerable own properties from one or more source objects to a
+ * target object. Returns the target object.
+ * @param target The target object to copy to.
+ * @param source1 The first source object from which to copy properties.
+ * @param source2 The second source object from which to copy properties.
+ */
+export function assign<T, U, V>(target: T, source1: U, source2: V): T & U & V;
```

Comment:
This is technically `assign(target, ...sources)`. Is there a way to express this in TS?